docs/src/components/Header.js

```diff
@@ -1,68 +1,102 @@
 // @flow
-import React from 'react';
-import { Box, Text, Icon, Link as GestaltLink } from 'gestalt';
+import React, { Component } from 'react';
+import { Box, Text, Icon, IconButton, Link as GestaltLink } from 'gestalt';
 import Link from './Link.js';
 
-export default function Header() {
-  return (
-    <Box
-      paddingY={2}
-      paddingX={4}
-      mdPaddingX={6}
-      lgPaddingX={8}
-      color="pine"
-      display="flex"
-      direction="row"
-      alignItems="center"
-    >
-      <Box marginStart={-2} marginEnd={-2}>
-        <Text size="lg" color="white" weight="bold">
-          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-          <Link to="/">
-            <Box padding={2}>
-              <Box
-                display="flex"
-                direction="row"
-                alignItems="center"
-                marginLeft={-1}
-                marginRight={-1}
-              >
-                <Box paddingX={1}>
-                  <Icon
-                    icon="pinterest"
-                    color="white"
-                    size={24}
-                    accessibilityLabel="Pinterest Logo"
-                  />
+type State = {|
+  isRTL: boolean,
+|};
+
+export default class Header extends Component<{}, State> {
```

Comment:
@selena-zeng please use a function with `useState` instead. We want to avoid adding new class components in Gestalt